main.go

```diff
@@ -14,6 +14,7 @@ import (
 
 var (
 	flagKubeConfigPath = flag.String("config", "", "Path of a kube config file, if not provided the app will try to get in cluster config")
+	flagResyncPeriod   = flag.Int("resync-period", 60, "Namespace watcher cache resync period in minutes. Defaults to 60")
```

Comment:
What about using `flag.Duration` which I believe calls `time.ParseDuration` and compehends things like `1s`, `30m`, `2h`?

https://golang.org/pkg/flag/#Duration
https://golang.org/pkg/time/#ParseDuration

Reply:
fancy! trying it now